pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,8 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		commit := fmt.Sprintf("%s", rev.Commit[:(len(rev.Commit)/3)])
```

Comment:
ser can send whatever, but sending something other than commit hash will probably fail the build and it is either an user error or something really nasty happened in between... 

I think the best thing we can do here is to validate the Commit field on sourceRevision (if we are not already doing that) and allow only SHA256 here (or empty string).

@bparees ?